tests/fiaas_deploy_daemon/deployer/kubernetes/deployment/test_secrets.py

```diff
@@ -49,26 +52,35 @@ def deployment():
 
 class TestSecrets(object):
     @pytest.fixture(params=[
-        # (Enable generic, Enable strongbox, app strongbox, Name of called mock)
-        (True, True, True, "generic_init_secrets"),
-        (True, True, False, "generic_init_secrets"),
-        (True, False, True, "generic_init_secrets"),
-        (True, False, False, "generic_init_secrets"),
-        (False, True, True, "generic_init_secrets"),
-        (False, True, False, "kubernetes_secrets"),
-        (False, False, True, "kubernetes_secrets"),
-        (False, False, False, "kubernetes_secrets"),
+        # (Enable generic, Enable strongbox, app strongbox, app-spec secrets, default_secret Name of called mock)
+        (True, True, True, False, False, "generic_init_secrets"),
+        (True, True, True, True, False, "generic_init_secrets"),
+        (True, True, False, False, False, "generic_init_secrets"),
+        (True, True, False, True, False, "generic_init_secrets"),
+        (True, False, True, False, False, "generic_init_secrets"),
+        (True, False, True, True, False, "generic_init_secrets"),
+        (True, False, False, False, False, "generic_init_secrets"),
+        (True, False, False, True, False, "generic_init_secrets"),
+        (False, True, True, False, False, "generic_init_secrets"),
+        (False, True, True, True, False, "generic_init_secrets"),
+        (False, True, False, False, False, "kubernetes_secrets"),
+        (False, True, False, True, False, "generic_init_secrets"),
+        (False, False, True, False, False, "kubernetes_secrets"),
+        (False, False, True, True, False, "generic_init_secrets"),
+        (False, False, False, False, False, "kubernetes_secrets"),
+        (False, False, False, True, False, "generic_init_secrets"),
```

Comment:
the fifth parameter/default_secret is always `False`. Is there a test case/cases missing, or does it make sense to remove the parameter? (Or am I missing something here?)

Reply:
I forgot to go back and change it, done with 4fcd3813433aba9ef05e7f5b3de158702d404e6e. Well spotted :)